src/github-types.ts

```diff
@@ -2,7 +2,9 @@ import { RestEndpointMethodTypes } from "@octokit/rest";
 
 export type GitHubIssue = RestEndpointMethodTypes["issues"]["get"]["response"]["data"];
 export type GitHubPullRequest = RestEndpointMethodTypes["pulls"]["get"]["response"]["data"];
-export type GitHubIssueComment = RestEndpointMethodTypes["issues"]["listComments"]["response"]["data"][0];
+export type GitHubIssueComment = RestEndpointMethodTypes["issues"]["listComments"]["response"]["data"][0] & {
+  isMinimized?: boolean;
```

Comment:
I'm surprised its not included in some type. Perhaps you can find the correct type?

Reply:
REST does not contain it, and GraphQL type has a different shape so I don't think the proper type exists. Maybe we should eventually switch to full GraphQl for this plugin.